Source/WTF/wtf/URL.cpp

```diff
@@ -250,11 +250,7 @@ String URL::fileSystemPath() const
     if (!isLocalFile())
         return { };
 
-    auto result = decodeEscapeSequencesFromParsedURL(path());
-#if PLATFORM(WIN)
-    result = FileSystem::fileSystemRepresentation(result);
-#endif
-    return result;
+    return decodeEscapeSequencesFromParsedURL(path());
```

Comment:
Since this is a general method - does this change work for all the users of this method?

Reply:
Further: Why do we need to remove this?

Reply:
I'm walking back this change for the moment and will deal with it in a separate commit